### PR TITLE
Django installation & models chapter through hemingwayapp

### PR DESCRIPTION
**Pull request by phalt**
I've put a few more chapters through hemingwayapp to help improve the grammar and wording.

It's hard finding a balance between too many pull requests and too many file changes, so if you have a preference let me know :laughing:

### CODE REVIEW

File: en/django_models/README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Django models
 
-What we want to create now is something that will store all posts in our blog. But to be able to do that we need to talk a little bit about things called `objects`.
+What we want to create now is something that will store all the posts in our blog. But to be able to do that we need to talk a little bit about things called `objects`.
 
 ## Objects
 
-There is a concept in programming called `Object-oriented programming`. The idea is that instead of writing everything as a boring sequence of programming instructions we can model things and define how they interact with each other.
+There is a concept in programming called `Object-oriented programming`. The idea is that instead of writing everything as a boring sequence of programming instructions we can model things and define how they interact with each other instead.
```
Comment (helenst) [inline en/django_models/README.md]: I'm not sure about having "instead" twice in this paragraph

File: en/django_models/README.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ What kind of things could be done with a blog post? It would be nice to have som
 
 So we will need a `publish` method.
 
-Since we already know what we want to achieve, we can start modeling it in Django!
+Since we already know what we want to achieve, lets start modeling it in Django!
```
Comment (helenst) [inline en/django_models/README.md]: should be let's

File: en/django_models/README.md [hunk 3]
```diff
@@ -142,13 +142,13 @@ Now we define the properties we were talking about: `title`, `text`, `created_da
...
+We will not explain every bit of code here since it would take too much time. You should take a look at Django's documentation if you want to know more about Model fields and how to define things other than those described above (https://docs.djangoproject.com/en/1.8/ref/models/fields/#field-types).
 
-What about `def publish(self):`? It is exactly our `publish` method we were talking about before. `def` means that this is a function/method and `publish` is the name of the method. You can change the name of the method, if you want. The naming rule is that we use lowercase and underscores instead of whitespaces (i.e. if you want to have a method that calculates average price you could call it `calculate_average_price`).
+What about `def publish(self):`? It is exactly the `publish` method we were talking about before. `def` means that this is a function/method and `publish` is the name of the method. You can change the name of the method, if you want. The naming rule is that we use lowercase and underscores instead of whitespaces. For example,  a method that calculates average price you could be called `calculate_average_price`.
```
Comment (helenst) [inline en/django_models/README.md]: "you could be called" should be either "you could call it" or just "could be called"

File: en/django_start_project/README.md [hunk 4]
```diff
@@ -9,12 +9,12 @@ is copyrighted by Markus Zapke-Gründemann et al.
...
+The names of some files and directories are very important for Django. You should not rename the files that we are about to create. Moving them to a different place is also not a good idea. Django needs to maintain a certain structure to be able to find important things.
 
-> Remember to run everything in the virtualenv. If you don't see a prefix `(myvenv)` in your console you need to activate your virtualenv. We explained how to do that in the __Django installation__ chapter in the __Working with virtualenv__ part. You can do that by typing the following command: `myvenv\Scripts\activate` on Windows or
-`myvenv/bin/activate` on Mac OS / Linux.
+> Remember to run everything in the virtualenv. If you don't see a prefix `(myvenv)` in your console you need to activate your virtualenv. We explained how to do that in the __Django installation__ chapter in the __Working with virtualenv__ part. Typing `myvenv\Scripts\activate` on Windows or
+`myvenv/bin/activate` on Mac OS / Linux will do it for you.
```
Comment (helenst) [inline en/django_start_project/README.md]: maybe "will do this for you"?

**Comment by phalt** [comment]
Thanks @helenst :yum:

**Comment by helenst** [comment]
Hope you don't mind all the nitpicks. I think these are nice changes to the way the text reads :cherries: :smile:

**Comment by phalt** [comment]
@helenst nope not at all, grammar changes need grammar checks :laughing:

**Comment by helenst** [comment]
Thank you for your work! :sparkles:

**Pull request merged**